AirTote/App.xaml.cs

```diff
@@ -6,6 +6,6 @@ public App()
 	{
 		InitializeComponent();
 
-		MainPage = new AppShell();
+		MainPage = new AirTote.TabMainPage();
```

Comment:
わざわざ`AirTote.TabMainPage`みたいにせずとも、同じnamespaceに属するので、下みたいな感じに書けます。

```suggestion
		MainPage = new TabMainPage();
```